prow/plugins/approve/approve.go

```diff
@@ -163,7 +164,7 @@ func handleGenericComment(log *logrus.Entry, ghc githubClient, oc ownersClient,
 		return err
 	}
 
-	opts := optionsForRepo(config, ce.Repo.Owner.Login, ce.Repo.Name)
+	opts := branchOptions(config, ce.Repo.Owner.Login, ce.Repo.Name, ce.Repo.DefaultBranch)
```

Comment:
It isn't appropriate to use the repos default branch. We need to use the branch that the PR is pointing to (`pr.Base.Ref`).